app/views/publish/courses/accredited_provider/new.html.erb

```diff
@@ -23,24 +23,26 @@
           <% if @provider.accredited_bodies.length > 0 %>
             <%= render partial: "provider_suggestion", collection: @provider.accredited_bodies.sort_by { |k| k[:provider_name] }, locals: { form: fields } %>
 
-            <div class="govuk-radios__divider">or</div>
+            <% unless FeatureService.enabled?(:accredited_provider_search) %>
```

Comment:
Makes sense, so we will remove the additional field when we remove the feature flag? Is it worth leaving a comment so we remember to remove it?

Reply:
Yup, you just left one on this PR 😂

Reply:
lol I meant in the code?

Reply:
![Screen Recording (Safari)](https://github.com/DFE-Digital/publish-teacher-training/assets/616080/fb3ef9d6-1d0a-48d4-b2f1-8c41ae72385c)

I also left it in the code 😜